web-ui/src/main/resources/catalog/components/utility/partials/hideshowpassword.html

```diff
@@ -0,0 +1,5 @@
+<span class="input-group-addon">
+  <span class="{{showHideClass}}"
```

Comment:
I'm not sure how bootstrap 3 handles buttons in input-group-addons, but I think using a `<button type="button">` element would be better here (semantically and for accessibility). If that's not possible using built-in boostrap classes then using the `span` is good enough I guess.

Reply:
Yes, adding a button instead of a `span` seems possible https://getbootstrap.com/docs/3.3/components/#input-groups-buttons